sdk/ml/azure-ai-ml/azure/ai/ml/entities/_compute/compute_instance.py

```diff
@@ -180,6 +180,7 @@ def __init__(
             location=kwargs.pop("location", None),
             resource_id=kwargs.pop("resource_id", None),
             description=description,
+            tags=kwargs.pop("tags", None),
```

Comment:
If you want tags to show up in intellisense, it needs to be an explicit keyword parameter in `__init__` (applies for all other compute subtype constructors)

Reply:
Added this to the compute subtype constructors.